spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/mongo/embedded/EmbeddedMongoAutoConfigurationTests.java

```diff
@@ -98,11 +98,7 @@ void customUnknownVersion() {
 
 	@Test
 	void customFeatures() {
-		EnumSet<Feature> features = EnumSet.of(Feature.TEXT_SEARCH, Feature.SYNC_DELAY, Feature.ONLY_WITH_SSL,
```

Comment:
The `ONLY_WITH_SSL` and `ONLY_WINDOWS_2008_SERVER ` features were dropped with `3.2.0`: https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/commit/3d3c0de90f076857976fc396aebff9aab90e8269